packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -121,8 +147,20 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
           </div>
         ) : null}
         {imageSrc || iconProps ? (
-          <div className={this._classNames.labelWrapper} style={{ maxWidth: imageSize.width * 2 }}>
-            {onRenderLabel!(props)}
+          <div
+            className={this._classNames.labelWrapper}
+            style={{
+              maxWidth: labelWrapperMaxWidth
+            }}
+          >
+            <div
+              ref={el => {
+                this._onLabelWrapperRef(el);
```

Comment:
Hmmm. Why not use createRef?

Reply:
Hmmm, how is it better?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/wiki/Component-Design#for-gathering-refs-avoid-string-refs-use-resolve-functions

Reply:
You only ever have 1 label. There is no need for an array.